src/content/docs/en/tutorial/4-layouts/2.mdx

```diff
@@ -131,14 +131,14 @@ Can you figure out what should go in the blanks so that the following two compon
     const { ___________ } = Astro.props
     ---
     <h1>{frontmatter.title}</h1>
-    <p>Written by:{frontmatter.______ } on {frontmatter.pubDate}</p>
+    <p>Written by: {frontmatter.______ } on {frontmatter.pubDate}</p>
     < _______ />
     <Footer />
     ```
     <details>
         <summary>Show the blanks filled in!</summary>
 
-        1.  ```markdown title="src/pages/posts/learning-astro.md" "../layouts" "pubDate:"
+        1.  ```markdown title="src/pages/posts/learning-astro.md" "layouts" "pubDate:"
```

Comment:
`../` is already included, so it shouldn't be highlighted.

![Screenshot 2023-07-16 at 17 04 18](https://github.com/withastro/docs/assets/7889778/6dd68795-9d24-449a-a56a-fc657da02ddb)

Reply:
Haha, I was about to make the same changes 😄 Now I don't have to anymore.

Can you also remove the `:` after `pubDate` (and the `.` before `author` below) from being highlighted?

I also noticed that the filename is `MarkdownLayout.astro`, but it is referenced as `MyMarkdownLayout.astro`. Does that fit into your PR as well?

Reply:
Hey @EEP-Benny, thanks for your review! I have corrected the lines you pointed out, so could you please check again?